makefile_sample/Makefile

```diff
@@ -35,6 +35,14 @@ fmt-ci:
 	fgt gofmt -l $(SOURCES)
 .PHONY: fmt-ci
 
+imports:
+	goimports -w $(SOURCES)
+.PHONY: imports
+
+imports-ci:
+	fgt goimports -w $(SOURCES)
```

Comment:
Also, could you update the README linters part? https://github.com/gonzaloserrano/go-nzalo/blob/master/makefile_sample/README.md